tests/api_connexion/schemas/test_dag_schema.py

```diff
@@ -30,7 +30,8 @@
 from airflow.configuration import conf
 from airflow.models import DagModel, DagTag
 
-SERIALIZER = URLSafeSerializer(conf.get('webserver', 'SECRET_KEY'))
+# the secret key is always available, so we can ignore Optional here
+SERIALIZER = URLSafeSerializer(conf.get('webserver', 'secret_key'))  # type: ignore[arg-type]
```

Comment:
It’s probably a good idea to make the secret key a pytest param. I’ll do that after this PR is in.

Reply:
I actually went ahead and changed it. It's sooo much better than dealing with the type ignores :)